src/plugins/kibana_utils/common/create_getter_setter.ts

```diff
@@ -20,7 +20,7 @@
 export type Get<T> = () => T;
 export type Set<T> = (value: T) => void;
 
-export const createGetterSetter = <T extends object>(name: string): [Get<T>, Set<T>] => {
+export const createGetterSetter = <T>(name: string): [Get<T>, Set<T>] => {
```

Comment:
This got in my way and I decided to just change it here - if there is a specific reason `T` always _has_ to extend object, I can roll that back. But IMHO putting numbers/booleans in here also makes perfect sense.